invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -95,13 +96,11 @@ def __init__(self):
                            'its relevance quickly.'
         }
         self.field_args['publisher'] = {
-            'hidden': True,
-            'value': self.publisher_default
-            # 'description':
-            # 'Here should be stored the site that will host the BE2Share ' +\
-            # 'container, so that in case of access problems, people can ' +\
-            # 'be contacted. This element can be created automatically ' +\
-            # 'dependent on the centre.'
+            'value': self.publisher_default,
```

Comment:
@emanueldima do you know how to handle the value so it isn't stored as `None` in the DB? It only occurs if the user empties the prefilled field.